egs/an4/asr2/conf/tuning/train_mtlalpha0.5.yaml

```diff
@@ -0,0 +1,40 @@
+# network architecture
+frontend-module: "espnet.nets.pytorch_backend.frontend_asr:FrontendASR"
```

Comment:
As this recipe is placed for testing of frontend recipe, WPE should also be tested. What's your opinion?